script/functions.sh

```diff
@@ -82,7 +84,32 @@ function run_specs_and_record_done {
     rspec_bin=script/rspec_with_simplecov
   fi;
 
-  $rspec_bin spec --backtrace --format progress --profile --format progress --out $SPECS_HAVE_RUN_FILE
+  $rspec_bin spec --backtrace --profile --format progress --out $SPECS_HAVE_RUN_FILE
```

Comment:
The duplication is intentional -- the first `--format progress` sets the formatter for stdout and then `--format progress --out $SPECS_HAVE_RUN_FILE` uses a second instance of the progress formatter that outputs to `$SPECS_HAVE_RUN_FILE`.